playbooks/roles/streisand-mirror/vars/tor.yml

```diff
@@ -12,40 +12,92 @@ tor_base_download_url: "https://dist.torproject.org/torbrowser"
 # Windows
 tor_browser_bundle_windows_filename: "torbrowser-install-{{ tor_browser_bundle_version }}_en-US.exe"
 tor_browser_bundle_windows_sig_filename: "{{ tor_browser_bundle_windows_filename }}.asc"
+
+tor_browser_bundle_windows_filename_fr: "torbrowser-install-{{ tor_browser_bundle_version }}_fr.exe"
```

Comment:
Ok. I'm happy to punt this to a separate issue for now since it's pretty theoretical when there's only one translation :-)